portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -20,11 +20,13 @@
   private final String lastName;
   private final String commentText;
   private final String date;
-
-  public Comment(String firstName, String lastName, String commentText, String date) {
+  private final String commentId;
+  
+  public Comment(String firstName, String lastName, String commentText, String date, String commentId) {
```

Comment:
I would call `commentId` just `id` since this type is `Comment`.

Reply:
changes made.